src/calculateRentalCost.js

```diff
@@ -4,7 +4,12 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  } else if (days < 7) {
```

Comment:
Avoid using `else if` after an `if` statement that contains a `return`. You can simply use an `if` statement since `return` will exit the function if the condition is met.